docs/releasenotes/5.1.0.rst

```diff
@@ -0,0 +1,36 @@
+5.1.0
+-----
+
+New File Format
+================
```

Comment:
The number of equals signs does not match the number of characters in the heading.